README.md

```diff
@@ -1,18 +1,36 @@
-# DTS Airflow stack
+# DTS Airflow
 
-This stack is used to run DTS ETL processes and the production instance is deployed on `atd-data03` at `/usr/airflow/atd-airflow`. Local development is available, and instructions are below.
+This repository hosts Data & Technology Service's [Airflow](https://airflow.apache.org/) code. We use Airflow as our primary orchestration platform for scheduling and monitoring our automated scripts and integrations.
+
+The production Airflow instance is available at `https://airflow.austinmobility.io/`. It requires COA network access.
+
+Our Airflow instance is hosted on `atd-data03` at `/usr/airflow/atd-airflow`. Local development is available, and instructions are below.
 
 The stack is composed of:
 
 - Airflow v2 ([Docker image](https://hub.docker.com/r/apache/airflow))
 - [HAProxy](https://www.haproxy.org/) to distribute HTTP requests over the stack
 - [Flower](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/security/flower.html) workers dashboard to monitor remote workers
 
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Developing a new DAG](#developing-a-new-dag)
+  - [Tags](#tags)
+  - [Moving to production](#moving-to-production)
+- [Utilities](#utilities)
+  - [1Password utility](#1password-utility)
+  - [Slack operator utility](#slack-operator-utility)
+- [Useful Commands](#useful-commands)
+- [Updating the stack](#updating-the-stack)
+- [HAProxy and SSL](#haproxy-and-ssl)
+  - [HAProxy operation](#haproxy-operation)
+
```

Comment:
❤️ 🚀 💯

Reply:
+1 - love the polish and the new additions about the new xcoms addition. 🙏

Reply:
credit where it's due, i have an extension that makes the TOC: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one